product_docs/docs/pem/8/installing_pem_agent/installing_on_linux/x86_amd64/index.mdx

```diff
@@ -25,6 +25,7 @@ For Linux x86-64 (amd64) see:
   - [CentOS 7](pem_agent_centos7_x86)
   - [SLES 15](pem_agent_sles15_x86)
   - [SLES 12](pem_agent_sles12_x86)
+  - [Ubuntu 22.04](pem_agent_ubuntu_22)
```

Comment:
When you click "Ubuntu 22.04" in the bulleted list, it shows a 404 page not found.

Reply:
fixed